test/benchmark/simulator_benchmark.py

```diff
@@ -225,7 +184,7 @@ def _run(self, runtime, app, measure, measure_count, noise_name, qubit):
             raise ValueError('no qobj: measure={0}:{1}, qubit={2}'.format(measure, measure_count, qubit))
 
         start = time()
-        result = simulator.run(qobj, noise_model=noise_model, **backend_options).result()
+        result = simulator._run(qobj, noise_model=noise_model, **backend_options)
```

Comment:
How come we need to change this?  It seems like it's skipping some normal set-up and wrapping code from the timed portion.

Reply:
I wanted to evaluate only C++ implementation as possible by generating `qobj` before running simulation.